modules/virt-schedule-supported-cpu-model-vms.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-schedule-vms.adoc
+
+[id="virt-schedule-supported-cpu-model-vms_{context}"]
+= Scheduling virtual machines with the supported CPU model
+
+You can configure a CPU model for a virtual machine (VM) or a virtual machine instance (VMI) to schedule it on a node where its CPU model is supported.
+
+.Procedure
+
+. Edit the `domain` spec of your virtual machine configuration file. The following example shows a specific CPU model defined for a VMI:
++
+[source,yaml]
+----
+apiVersion: kubevirt.io/v1alpha3
```

Comment:
Please use this example instead, it fixes indentation:
```
apiVersion: kubevirt.io/v1alpha3
kind: VirtualMachineInstance
metadata:
  name: myvmi
spec:
  domain:
    cpu:
      model: Conroe <1>
```

Reply:
Fixed indentation as discussed in gchat.

Reply:
Fixed.